ios/FluentUI/Vnext/Divider/Divider.swift

```diff
@@ -64,38 +63,38 @@ public struct FluentDivider: View {
                                   leading: tokens.padding,
                                   bottom: 0,
                                   trailing: tokens.padding))
-            .designTokens(tokens,
-                          from: theme,
-                          with: windowProvider)
+            .resolveTokens(self)
+            .resolveTokenModifier(self, value: state.orientation)
     }
+
+    public func overrideTokens(_ tokens: DividerTokens?) -> FluentDivider {
+        state.overrideTokens = tokens
+        return self
+    }
```

Comment:
Seems like this is usually in a *Modifiers.swift file, but I wasn't really sure what the point of putting it in a file all by itself was.

Reply:
I think it's probably still worth making the new file (A) for consistency, and (B) because there will more modifiers when the control is ready to be hosted in SwiftUI, specifically for `color` and `orientation`.